src/components/SearchArticles.tsx

```diff
@@ -1,9 +1,10 @@
 import { Link } from "react-router-dom";
+import { ArticlesType } from "../types/models/types";
 
-function SearchArticles({ articles }) {
+function SearchArticles({ articles }: ArticlesType[] | any) {
```

Comment:
Could you use a more specific type than any as the union with any might disable type checking?